### PR TITLE
qemuhck: Isolate swtpm_localca state

### PR DESCRIPTION
**Pull request by akihikodaki**
The default swtpm_localca state may not be accessible so create our own.

### CODE REVIEW

File: etc/swtpm_setup.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+create_certs_tool=swtpm_localca
+create_certs_tool_config=etc/swtpm-localca.conf
```
Comment (kostyanf14) [inline etc/swtpm_setup.conf:2]: do we need create_certs_tool_options or this is optional?

File: etc/swtpm-localca.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+statedir=${AUTOHCK_STATEDIR}
+signingkey=${AUTOHCK_STATEDIR}/signkey.pem
+issuercert = ${AUTOHCK_STATEDIR}/issuercert.pem
```
Comment (kostyanf14) [inline etc/swtpm-localca.conf:3]: Do we need certserial or this is optional?

**Review by kostyanf14 — COMMENTED** [review]
Looks good, just have questions about the config after compared with the original /etc files.

File: etc/swtpm_setup.conf (hunk 1 above)
Comment (akihikodaki) [reply etc/swtpm_setup.conf]: It is necessary. `etc/swtpm-localca.conf` contains paths to files that must be relocated to the workspace.

Comment (akihikodaki) [reply etc/swtpm_setup.conf]: You mentioned `create_certs_tool_options`. I confirmed it works without the parameter.

File: etc/swtpm-localca.conf (hunk 2 above)
Comment (akihikodaki) [reply etc/swtpm-localca.conf:3]: It is optional.

**Review by kostyanf14 — APPROVED** [review]

**Pull request merged**